pkg/test/resourcefixture/testdata/resourceoverrides/computeinstance/networkipcomputeinstance/create.yaml

```diff
@@ -32,6 +32,6 @@ spec:
   - subnetworkRef:
       name: default
     networkIp: "10.128.0.9"
-    accessConfigs:
+    accessConfig:
```

Comment:
Can we change a beta field?  I'm curious how this is not captured before

Reply:
I do not think we have schema checks in samples/dynamic tests yet(?) 
Here's an open issue: https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1201